packages/changelog-bot/src/main/ChangelogBot.ts

```diff
@@ -81,11 +85,17 @@ class ChangelogBot {
     const omittedMessage = '... (content omitted)';
 
     const changelog = await Changelog.generate({
-      exclude: ['build', 'chore', 'docs', 'refactor', 'test'],
+      exclude: ChangelogBot.SETUP.EXCLUDED_COMMIT_TYPES,
       repoUrl: `https://github.com/${repoSlug}`,
       tag: previousGitTag,
     });
 
+    if (!changelog.match(listItems)) {
+      const excludedTypes = ChangelogBot.SETUP.EXCLUDED_COMMIT_TYPES.join(', ');
+      const errorMessage = `Could not generate a meaningful changelog from the commit types given (excluded ${excludedTypes})`;
```

Comment:
```suggestion
      const errorMessage = `Could not generate a meaningful change-log from the commit types given (excluded ${excludedTypes}).`;
```

Since it's start with a capital.. maybe a `.` in the end :)

Reply:
<img width="213" alt="screenshot 2018-10-26 at 17 37 07" src="https://user-images.githubusercontent.com/125676/47573360-cbcd4b80-d945-11e8-8d76-984205e87542.png">

Reply:
https://en.wikipedia.org/wiki/Changelog

Reply:
https://en.wiktionary.org/wiki/changelog

Reply:
https://developer.microsoft.com/en-us/graph/docs/concepts/changelog

Reply:
http://www.yourdictionary.com/changelog

Reply:
https://mozilla.github.io/meao/2018/03/08/mdn-changelog/

Reply:
:P then just a dot :)